GenderPayGap.WebUI/Views/Viewing/Finder/Parts/Facets/Keywords.cshtml

```diff
@@ -28,22 +25,25 @@
         <div class="gov-uk-c-searchbar">
             <div class="gov-uk-l-searchbar__table">
                 <div class="gov-uk-l-searchbar__cell">
-                    <input id="FinderKeywords"
-                           value="@(Model.search)"
-                           aria-expanded="false"
-                           class="gov-uk-c-searchbar__input"
-                           name="search"
-                           type="text"
-                           autocomplete="off"
-                           spellcheck="false"
-                           maxlength="100"/>
+                    <label id="FinderKeywordsId" for="FinderKeywords" class="visuallyhidden">@(!string.IsNullOrEmpty(Model.t) && Model.t == "1" ? "Search by employer name" : "Search query")</label>

```

Comment:
This logic handles the initial case before a button is clicked

Reply:
A few things:
* This line is quite long - I'd split it out over a few lines
* I'd make the options `Search by employer name` or `Search by industry sector` (or something like that) - "Search query" seems a bit vague
* I'd give the element a more specific id - perhaps `SearchInputLabel`

Reply:
Btw, I know this isn't your code originally, so you didn't choose the ID, etc.
With infinite time, we could make loads of improvements, but these few seem worth making now.

Reply:
- I've shortened all the line lengths in this file to make it a bit tidier
- I've replaced the label ID with `SearchInputLabel`
- For the options, because of the initial logic `!string.IsNullOrEmpty(Model.t) && Model.t == "1"` I don't think a user can ever end up in the second conditional result. So I don't expect them to ever be able to see the "search query" label. I have just left it in as an intentionally generic label in the case that `Model.t` is undefined. If js is enabled then the label will be set to "Search by employer type or SIC code".